web/public/js/controllers/user-profile-controller.js

```diff
@@ -27,6 +27,7 @@ function cdUserProfileCtrl($scope, $rootScope, $state, $window, auth, cdUsersSer
   $scope.editMode = false;
   $scope.publicMode = false;
   $scope.publicChampion = false;
+  $scope.loggedInUserIsMemberOfDojoChampion = false;
```

Comment:
Nope, if the profile being viewed is a champion for the dojo that the user seeing is belonging to